### PR TITLE
Magento_CheckoutAgreements: avoid using deprecated escape* methods fr…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Static Tests, Semantic Version Checker, Database Compare

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

File: app/code/Magento/CheckoutAgreements/view/frontend/templates/multishipping_agreements.phtml [hunk 1]
```diff
@@ -8,7 +8,9 @@ use Magento\CheckoutAgreements\Model\AgreementModeOptions;
 ?>
 <?php
 /**
+ *
  * @var $block \Magento\CheckoutAgreements\Block\Agreements
+ * @var \Magento\Framework\Escaper $escaper
  * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
```
Comment (ihor-sviziev) [inline app/code/Magento/CheckoutAgreements/view/frontend/templates/multishipping_agreements.phtml]:
```suggestion
/**
 * @var \Magento\CheckoutAgreements\Block\Agreements $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
```

File: app/code/Magento/CheckoutAgreements/view/frontend/templates/agreements.phtml [hunk 2]
```diff
@@ -8,7 +8,9 @@ use Magento\CheckoutAgreements\Model\AgreementModeOptions;
 ?>
 <?php
 /**
+ *
  * @var $block \Magento\CheckoutAgreements\Block\Agreements
+ * @var \Magento\Framework\Escaper $escaper
  */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/CheckoutAgreements/view/frontend/templates/agreements.phtml]:
```suggestion
/**
 * @var \Magento\CheckoutAgreements\Block\Agreements $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by sergeynezbritskiy** [comment]
Hi @ihor-sviziev I've applied your suggestions, but somehow the label `Progress: needs update` was not removed

File: app/code/Magento/CheckoutAgreements/view/frontend/templates/multishipping_agreements.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/CheckoutAgreements/view/frontend/templates/multishipping_agreements.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Database Compare

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Database Compare,

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Database Compare, Integration Tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Pull request closed without merging**